src/fake_useragent/utils.py

```diff
@@ -55,19 +55,10 @@ def find_browser_json_path() -> Path:
         return Path(str(file_path))
     except Exception as exc:
         logger.warning(
-            "Unable to find local data/jsonl file using importlib-resources. Try pkg-resource.",
+            "Unable to find local data/jsonl file using importlib-resources.",
```

Comment:
Unless the file was manually removed, `importlib.resources` should always be able to locate it. Thus this should not raise normally.

_I assume the package is always installed in the first place, even editable. Without that, I'm not sure even `pkg_resources` would work._